app/views/course/assessment/assessments/_assessment.html.slim

```diff
@@ -3,7 +3,9 @@
     - unless assessment.published?
       span title=draft_message(assessment)
         => fa_icon 'ban'.freeze
-    = render partial: 'course/lesson_plan/items/personal_time_boolean_icons', locals: { item: assessment }
+    - unless current_course_user && \
```

Comment:
How about `if current_course_user&.staff? && current_course_user&.timeline_algorithm != 'fixed'`?

I think `if` is easier to read than `unless`, and there's already a function which checks for staff roles.

Reply:
Oh just realised you can do `current_course_user.student?`. `unless current_course_user.student? && current_course_user&.timeline_algorithm == 'fixed'` is fine then.

Reply:
Ok done!